keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/handlers/DeliveryConfigNotificationHandler.kt

```diff
@@ -0,0 +1,59 @@
+package com.netflix.spinnaker.keel.slack.handlers
+
+import com.netflix.spinnaker.keel.notifications.NotificationType.DELIVERY_CONFIG_CHANGED
+import com.netflix.spinnaker.keel.slack.SlackConfigNotification
+import com.netflix.spinnaker.keel.slack.SlackService
+import com.slack.api.model.kotlin_extension.block.withBlocks
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+/**
+ * Sends notifications for delivery config updates (create, modify)
+ */
+@Component
+class DeliveryConfigNotificationHandler(
+  private val slackService: SlackService,
+  private val gitDataGenerator: GitDataGenerator
+) : SlackNotificationHandler<SlackConfigNotification> {
+  override val supportedTypes = listOf(DELIVERY_CONFIG_CHANGED)
+
+  private val log by lazy { LoggerFactory.getLogger(javaClass) }
+
+  override fun sendMessage(notification: SlackConfigNotification, channel: String) {
+    with(notification) {
+      log.debug("Sending config changed notification for application ${notification.application}")
+      val imageUrl = "https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/assets/images/md_icons/config_change.png"
```

Comment:
Just a small suggestion, since these appear in multiple places now: in a future PR, we could create a `IconRepository` class coupled with a well defined enum of icon types to make this cleaner. So you'd end up with something like this here:
```kotlin
val imageUrl = iconRepository.getUrlFor(DELIVERY_CONFIG_CHANGED)
```
Another reason I'm suggesting this is that @ranihorev has proposed that the backend starts returning icons for resources and other things so that we don't have to change deck every time there's a new thing to represent.

Reply:
Or it could it even be just config in `keel.yml`.

Reply:
I'll think about this in the future either when I need to make another icon change or when we update the API :P thanks for the suggestion